### DISCUSSION TITLE
Import data is always pending on the cluster and asking all the memory available on the server

### DISCUSSION THREAD

**Discussion by xingzhe1129** [nextpyp/.github#25] [Q&A] [answered]
Hello, everyone,
I met an issue when I try to import tutorial data into nextPYP. The import data step is always pending on the cluster although there are GPUs available on the server. And then we found that nextPYP job is asking all memory available on the server, but there are some others task running on the same server. We can't make all memory on the server to be available for nextPYP. But we can import data and process data in nextPYP  on our standalone workstation. Does anyone met this issue? How can we solve this issue? Thank you so much in advance for any advices.

**Comment by abartesaghi** [comment, marked as answer]
nextPYP allows you to configure how many threads and how much memory to use for each specific task, but you are right, the import data block does not currently allow you to specify these resources. This was not by design, so we will fix this in the upcoming release. The import data job is very lightweight and should not need very much RAM or CPU, and definitely not a GPU. Thanks for reporting this issue.

**Comment by xingzhe1129** [comment]
Thank you so much for fixing this in the upcoming release.
